public/app/plugins/datasource/jaeger/_importedDependencies/types/trace.ts

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
```

Comment:
I wonder can should we move this file to the o11y-ds-frontend and share with Tempo? It's a 1:1 match with Tempo. Honestly not 100% certain this is ideal but what do you think?

Reply:
Hmm which file are you talking about? I can't find a similar one in Tempo. But I see that Tempo imports from a `trace.ts` file in `@grafana/data`, but I don't know if they are actually the same file—they are not a 1:1 match, but maybe there are just syntactic differences 🤔

Reply:
`grafana/public/app/features/explore/TraceView/components/types/trace.ts` seems to be the same file. In any case, we will can be over time removing from _importedDependencies like previously agreed.